src/main/java/com/ebivariation/contigalias/entities/AssemblyEntity.java

```diff
@@ -57,71 +58,100 @@ public long getId() {
+    public AssemblyEntity setOrganism(String organism) {
+        this.organism = organism;
+        return this;
     }
 
-    public String getRefseq() {
-        return refseq;
+    public long getTaxid() {
+        return taxid;
     }
 
-    public boolean isGenbankRefseqIdentical() {
-        return isGenbankRefseqIdentical;
+    public AssemblyEntity setTaxid(long taxid) {
+        this.taxid = taxid;
+        return this;
     }
 
-    public List<ChromosomeEntity> getChromosomes() {
-        return chromosomes;
+    public String getGenbank() {
+        return genbank;
     }
 
-    public AssemblyEntity setId(long id) {
-        this.id = id;
+    public AssemblyEntity setGenbank(String genbank) {
+        this.genbank = genbank;
         return this;
     }
 
-    public AssemblyEntity setName(String name) {
-        this.name = name;
-        return this;
+    public String getRefseq() {
+        return refseq;
     }
 
-    public AssemblyEntity setOrganism(String organism) {
-        this.organism = organism;
+    public AssemblyEntity setRefseq(String refseq) {
+        this.refseq = refseq;
         return this;
     }
 
-    public AssemblyEntity setTaxid(long taxid) {
-        this.taxid = taxid;
-        return this;
+    public boolean isGenbankRefseqIdentical() {
+        return isGenbankRefseqIdentical;
     }
 
-    public AssemblyEntity setGenbank(String genbank) {
-        this.genbank = genbank;
+    public AssemblyEntity setGenbankRefseqIdentical(boolean genbankRefseqIdentical) {
+        isGenbankRefseqIdentical = genbankRefseqIdentical;
         return this;
     }
 
-    public AssemblyEntity setRefseq(String refseq) {
-        this.refseq = refseq;
-        return this;
+    public List<ChromosomeEntity> getChromosomes() {
+        return chromosomes;
     }
 
     public AssemblyEntity setChromosomes(List<ChromosomeEntity> scaffolds) {
         this.chromosomes = scaffolds;
         return this;
     }
 
-    public AssemblyEntity setGenbankRefseqIdentical(boolean genbankRefseqIdentical) {
-        isGenbankRefseqIdentical = genbankRefseqIdentical;
-        return this;
+    @Override
+    public String toString() {
+        StringBuilder builder = new StringBuilder();
+        builder.append("Name :\t")
+               .append(this.name)
+               .append("\n")
+               .append("Organism :\t")
+               .append(this.organism)
+               .append("\n")
+               .append("Tax ID :\t")
+               .append(this.taxid)
+               .append("\n")
+               .append("Genbank :\t")
+               .append(this.genbank)
+               .append("\n")
+               .append("Refseq :\t")
+               .append(this.refseq)
+               .append("\n")
+               .append("Genbank & Refseq identical :\t")
+               .append(isGenbankRefseqIdentical)
+               .append("\n");
+        if (this.chromosomes != null) {
+            builder.append("No of chromosome :\t")
```

Comment:
```suggestion
            builder.append("Number of chromosome :\t")
```
it doesn't make sense to save 4 bytes at the cost of being less understandable.